Python/rerf/RerF.py

```diff
@@ -79,7 +85,21 @@ def fastRerF(
 
     forestClass = pyfp.fpForest()
 
-    forestClass.setParameter("forestType", forestType)
+    # Set forestType with additional parameters specific to that type
+    if forestType == "binnedBaseTern":
+        forestClass.setParameter("methodToUse", 1)
+    elif forestType == "S-RerF":
+        forestClass.setParameter("forestType", "binnedBaseTern")
+        forestClass.setParameter("methodToUse", 2)
+        forestClass.setParameter("imageHeight", imageHeight)
+        forestClass.setParameter("imageWidth", imageWidth)
+        forestClass.setParameter("patchHeightMax", patchHeightMax)
+        forestClass.setParameter("patchHeightMin", patchHeightMin)
+        forestClass.setParameter("patchWidthMax", patchWidthMax)
```

Comment:
add checking here to make sure min isn't > max and min is > 1 and max <= height/width

Reply:
We check those things in C++ ...